CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+<!-- Released section -->
+<!-- Don't change this section unless doing release -->
+
+## [1.30.0/0.52.0/0.6.0/0.0.9] 2024-09-09
+
+This release drops the compatibility guarantee of [Go 1.21].
```

Comment:
```suggestion
This release drops the compatibility guarantee for [Go 1.21].
```

This seems duplicative of the Removed section entry. Is it needed?

Reply:
I am ok to drop this sentence.

Reply:
Done